config/src/config/consensus_config.rs

```diff
@@ -429,21 +412,23 @@ impl ConsensusConfig {
                 config.max_sending_block_txns,
                 "txns".to_string(),
             ),
+            (
+                config.quorum_store.receiver_max_batch_txns as u64,
+                config.max_sending_block_txns_after_filtering,
+                "txns".to_string(),
+            ),
+            (
+                config.quorum_store.receiver_max_batch_txns as u64,
+                config.min_max_txns_in_block_after_filtering_from_backpressure,
+                "txns".to_string(),
+            ),
             (
                 config.quorum_store.receiver_max_batch_bytes as u64,
                 config.max_sending_block_bytes,
                 "bytes".to_string(),
             ),
         ];
         for backpressure_values in &config.pipeline_backpressure {
-            recv_batch_send_block_pairs.push((
-                config.quorum_store.receiver_max_batch_txns as u64,
-                backpressure_values.max_sending_block_txns_after_filtering_override,
-                format!(
-                    "backpressure {} ms: txns",
-                    backpressure_values.back_pressure_pipeline_latency_limit_ms,
-                ),
-            ));
```

Comment:
we don't require backpressure to be at least the batch size any more, only min_max_txns_in_block_after_filtering_from_backpressure